.gitignore

```diff
@@ -5,3 +5,38 @@ none
 config.yaml
 Save Backups/
 FFXCutsceneRemover.exe
+tas_saves/ffx_000
```

Comment:
We very much, do not want to ignore these files.

Reply:
Fixed gitignore.